rules/sonic-py-common.mk

```diff
@@ -4,7 +4,8 @@ ifeq ($(ENABLE_PY2_MODULES), y)
     SONIC_PY_COMMON_PY2 = sonic_py_common-1.0-py2-none-any.whl
     $(SONIC_PY_COMMON_PY2)_SRC_PATH = $(SRC_PATH)/sonic-py-common
     $(SONIC_PY_COMMON_PY2)_DEPENDS += $(SWSSSDK_PY2)
-    $(SONIC_PY_COMMON_PY2)_PYTHON_VERSION = 2
+    $(SONIC_PY_COMMON_PY2)_DEBS_DEPENDS += $(PYTHON_SWSSCOMMON)
```

Comment:
>PYTHON_SWSSCOMMON [](http://example.com/codeflow?start=45&length=17)

Thanks for fixing the dependency! It is not related to front panel port prefix. Could you raise a separate PR for the fix? We may need to backport the fix.

Reply:
The reason I have to add the dependency here is because I needed the new port panel regex from swsscomon in sonic_py_common, if you think it's worth to make it a separate PR I can do it.

Reply:
The dependency fix itself is a good bug fix. Please help.

Reply:
Sure, no problem, i'll create a new PR with only the dependency fix and send it to you.

Reply:
Actually this was fixed in master few weeks ago by - https://github.com/Azure/sonic-buildimage/commit/e4ed475fcde0d68abb3b2814a71b76296642a7ee.
I just took the latest master and removed my change.